lang/java/BUILD.md

```diff
@@ -44,6 +46,12 @@ To perform "fast" build, which skips tests and all code quality enforcement tool
 
     mvn clean install -DskipTests -TC1 -P!code-quality
 
+
+  * Please follow the following wiki for all running all java builds [Java Build Wiki](https://cwiki.apache.org/confluence/display/REEF/Linux), the REEF unit tests require a number of open files which is greater than the default open file limit on a number of Linux distributions such as Ubuntu 16.04/16.10.  This limit is controlled in the shell by the "ulimit -n" command
```

Comment:
Major versions should be fine. That said, the link to the Wiki seems troubling: Is there anything on the wiki that *should* be in the build instructions? If so, I'd rather have that information all in these build instructions to make them self-contained.